site/docs/index.md

```diff
@@ -0,0 +1,17 @@
+---
+hasFrame: true
+---
+
+# Library Example
+
+A library that does nothing.
+
+### A fancy landing page for {{name}}
```

Comment:
Where is `name` set?

Reply:
Nowhere unfortunately. You can see [here](https://vincenzobaz.github.io/library-example/docs/index.html) that no `name` ends up in the render. This happens for two reasons

 1. No `name` is set afaik
 2. Liquid variables (or any liquid syntax) is not accessible from markdown files.

The solution will be to inject it in `mdocVariables`

Reply:
3. Remove it?

```suggestion
### A fancy landing page for library example
```

Reply:
writing two words instead of modifying the entire build to inject a variable? :scream: